File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        const s = new Set();
        const len = nums.length;

        for (let i = 0; i < len; i++) {
            s.add(nums[i]);
        }

        return len !== s.size;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        res = defaultdict(int)
        for i in range(len(nums)):
            res[nums[i]] += 1
        
        sorted_items = sorted(res.items(), key = lambda item: item[1], reverse = True)

        return [item[0] for item in sorted_items[:k]]